development/includes/navbar.pug

```diff
@@ -0,0 +1,26 @@
+
```

Comment:
Looks like this whole file is extra and duplicate to the the _top-nav.pug file. Delete this one.